tuned_lens/scripts/ingredients.py

```diff
@@ -115,15 +118,18 @@ def load_tokenizer(self, must_use_cache: bool = False) -> PreTrainedTokenizerBas
         return tokenizer
 
     def load(
-        self, must_use_cache: bool = False
+        self, device: Optional[th.device], must_use_cache: bool = False
```

Comment:
Document what device does in the doc string.

Reply:
done